plugins/blip/basics/chartbasicsfactory.js

```diff
@@ -70,7 +70,7 @@ var BasicsChart = React.createClass({
   componentWillMount: function() {
     var basicsData = this.props.patientData.basicsData;
     if (basicsData.sections == null) {
-      basicsData = _.assign(basicsData, basicsState);
+      basicsData = _.assign({}, basicsData, _.cloneDeep(basicsState));
```

Comment:
Something for considersation now that Object.assign() exists is that we have a choice between _.assign and it. I don't think it is majorly important, but potentially good for consistency to choose one over the other now that we use ES6/babel.